WalletWasabi.Tests/WalletWasabi.Tests.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="AltCover" Version="5.3.675" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+    <PackageReference Include="NBitcoin.TestFramework" Version="1.7.6" />
```

Comment:
Please don't add packages to the project.

Reply:
I tried to use Wasabi's `CoreNode` from Wasabi.Tests instead but its design make it impossible to use properly in my case. Mainly due to the use of the Test Global and the fact there is no way to separate the node creation (`Create`, which should not start the node) and the start of the node.

Would you agree for either: 
* Removing CoreNode from Wasabi.Tests and use NBitcoin.TestFramework instead everywhere.
* or let me refactor the Wasabi one so that we don't end up with singletons dependencies.